ci/cases/yamls/atmaerosnowDA_defaults_ci.yaml

```diff
@@ -3,4 +3,4 @@ defaults:
 base:
   DOIAU: "NO"
   DO_JEDISNOWDA: "YES"
-  ACCOUNT: {{ 'SLURM_ACCOUNT' | getenv }}
```

Comment:
UFS-weather-model uses something named `ACCNR`.
Can we use that instead of `BATCH_ACCOUNT` to keep some semblance of consistency?